### PR TITLE
Several fixes

### PR DESCRIPTION
**Pull request by fernandog**
@p0psicles @ratoaq2 why have to be this way in the first commit?

### CODE REVIEW

File: medusa/cache.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ def __init__(self, filename):
 
         :param filename:
         """
+        super(MutexLock, self).__init__(filename)
```
Comment (fernandog) [inline medusa/cache.py]: File "/home/osmc/Medusa/medusa/cache.py", line 20, in __init__
    super(MutexLock, self).__init__(filename)
  File "/home/osmc/Medusa/lib/dogpile/cache/backends/file.py", line 300, in __init__
    raise NotImplementedError()
NotImplementedError

File: medusa/server/api/v1/core.py [hunk 2]
```diff
@@ -254,8 +254,9 @@ class ApiCall(ApiHandler):
 
     _help = {"desc": "This command is not documented. Please report this to the developers."}
 
-    def __init__(self, args, kwargs):
+    def __init__(self, *args, **kwargs):
         # missing
+        super(ApiCall, self).__init__(*args, **kwargs)
```
Comment (fernandog) [inline medusa/server/api/v1/core.py]: @ratoaq2 got this:

```
2016-11-27 07:25:15 ERROR    TORNADO :: [782bb04] API :: 'tuple' object has no attribute 'ui_methods'
Traceback (most recent call last):
  File "/home/osmc/Medusa/medusa/server/api/v1/core.py", line 119, in get
    out_dict = _call_dispatcher(args, kwargs)
  File "/home/osmc/Medusa/medusa/server/api/v1/core.py", line 188, in call_dispatcher
    cur_out_dict = func(cur_args, cur_kwargs).run()  # call function and get response
  File "/home/osmc/Medusa/medusa/server/api/v1/core.py", line 2251, in __init__
    ApiCall.__init__(self, args, kwargs)
  File "/home/osmc/Medusa/medusa/server/api/v1/core.py", line 259, in __init__
    super(ApiCall, self).__init__(*args, **kwargs)
  File "/home/osmc/Medusa/medusa/server/api/v1/core.py", line 86, in __init__
    super(ApiHandler, self).__init__(*args, **kwargs)
  File "/home/osmc/Medusa/lib/tornado/web.py", line 172, in __init__
    application.ui_methods.items())
AttributeError: 'tuple' object has no attribute 'ui_methods'
```

**Comment by fernandog** [comment]
@ratoaq2 please? can you check those two comments i tagged you?

File: medusa/indexers/tmdb/tmdb.py [hunk 3]
```diff
@@ -36,8 +36,8 @@ def log():
 class Tmdb(BaseIndexer):
     """Create easy-to-use interface to name of season/episode name.
 
-    >>> t = tmdb()
-    >>> t['Scrubs'][1][24]['episodename']
+    t = tmdb()
+    t['Scrubs'][1][24]['episodename']
```
Comment (p0psicles) [inline medusa/indexers/tmdb/tmdb.py]: >>> is generically used as example lines. As in, this can be used on command line.

Comment (fernandog) [inline medusa/indexers/tmdb/tmdb.py]: @p0psicles pycharm gave error about tmdb()

**Comment by fernandog** [comment]
No one interested

**Comment by fernandog** [comment]
approve? removed the class commit

**Pull request merged**